integration_tests/conftest.py

```diff
@@ -74,4 +74,16 @@ def dev_session():
     process = spawn_process(command)
     time.sleep(5)
     yield
-    kill_process(process)
\ No newline at end of file
+    kill_process(process)
+
+@pytest.fixture(scope="session")
+def test_session():
+    os.environ["ROBYN_URL"] = "127.0.0.1"
+    os.environ["ROBYN_PORT"] = os.environ["PORT"]
```

Comment:
Try setting `ROBYN_URL` and `ROBYN_PORT` directly through `load_vars()`?

Reply:
but shouldn't that be left to the user, it won't always be the case that a user will provide a PORT and a URL in robyn.env  but even if he did those vars will be loaded automatically and he can just assign them to a variable and use them

Reply:
but that wouldn't be the case for other variables such as token or db credentials so should just auto invoke the variables that are relevant for robyn and other variables get left for the user to decide what to do with them?

Reply:
No no. What I mean is if you set `ROBYN_PORT` in the test then it will be automatically present in the environment. That way you don't need to set that variable in the fixture.

If that still doesn't make sense then we can hop on a quick call.

Reply:
i think i understand what you mean i will get on it and check with you afterwards

Reply:
Perfect!

Reply:
i think this should be it, I changed the content written in robyn.env to include `ROBYN_PORT` and `ROBYN_URL` and it worked, i also deleted the fixture i created in conftest.py since it is of no use now.